### DISCUSSION TITLE
blog/2022-02-04-raspberry-pi-4-usb-c-fallout

### DISCUSSION THREAD

**Discussion by giscus** [crazychenz/vinnie.work#39] [vinnie.work Comments]
# blog/2022-02-04-raspberry-pi-4-usb-c-fallout

As part of the Embedded Systems Analysis material development I was recently doing some work with a Raspberry Pi I had sitting on my desk. I noticed that the USB-C power connection had fallen out of the device. Naturally I got under my desk to fetch the squirrel-y cable and plug it back in. I got the cable all the way to the RaspberryPi before I realized that the connector that was supposed to be attached to the RPi was still on the cable. \_The whole USB-C assembly had cleanly popped of the RPi4.

https://www.vinnie.work/blog/2022-02-04-raspberry-pi-4-usb-c-fallout

**Comment by Aimedweevil** [comment]
Life saver. I had a RPI4 connected to a LCD and the USB-C connector that came with the screen made a short. using your instructions brought my RPi4 back to life. Thank you for your post.
